src/main/java/org/folio/marc4ld/service/ld2marc/field/impl/Bibframe2MarcFieldRuleImpl.java

```diff
@@ -57,9 +57,13 @@ public Bibframe2MarcFieldRuleImpl(
   }
 
   @Override
-  public boolean isSuitable(Resource resource, PredicateDictionary predicate) {
-    return Objects.equals(fieldRule.getTypes(), resource.getTypeNames())
-      && (isNull(predicate) || predicate.name().equals(fieldRule.getPredicate()));
+  public boolean isSuitable(ResourceEdge edge) {
+    var source = edge.getSource();
+    var target = edge.getTarget();
+    var predicate = edge.getPredicate();
+    return Objects.equals(fieldRule.getTypes(), target.getTypeNames())
+      && (isNull(predicate) || predicate.name().equals(fieldRule.getPredicate()))
+      && (isTypeLikeRuleParent(source) || isTypeLikeRuleParent(target));
```

Comment:
`|| isTypeLikeRuleParent(target)` is not needed here right? 
We need to compare the type of the source only with field rule's parent.

Reply:
Thanks! 
There was a misunderstanding of the setting